iidm/iidm-xml-converter/src/test/resources/V1_0/eurostag-tutorial-example1-geographicalTags.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why did you create this file? What is the difference with eurostag-tutorial-example1.xml? You can reuse it in the test.

Reply:
I did this because during the import we can read geographicalTags at their previous place and in properties, but we only export them in properties, so we coudn't make a round trip test

Reply:
I did not notice it but the eurostag-tutorial-example1.xml files in **previous** versions should not be modified (only the file in **current** version should be modified). This should be reverted and you will be able to use them for this test.

Reply:
If i do this, i will have to modify all previous round trip test because they will not pass.

Reply:
This is weird... There should be two cases:
- either your round trip reads in previous versions and writes in current versions => geographicalTags is read as an attribute and written as a property
- or your round trip reads in previous version and writes in previous versions => geographicalTags is read and written as an attribute

I don't see in which cases it breaks the round trip tests?